webpack/app/js/tile.js

```diff
@@ -69,4 +65,47 @@ export default class AgendaTile {
       },
     });
   }
+  updateDayPicker(data) {
+    this.datepicker.$day.html('');
+    for (let day of data) {
+      let cssclass = ['day'];
+      if (day.isSelected) {
+        cssclass.push('is-selected');
+      }
+      let $day = $(`
+        <li data-day="${day.datetime}" class="${cssclass.join(' ')}">
+          <div class="daypicker-day">${day.day}</div>
+          <div class="daypicker-weekday">${day.weekday}</div>
+        </li>
+      `);
+      if (day.hasAppointment) {
+        $day.addClass('has-appointment');
+      }
+      this.datepicker.$day.append($day);
+    }
+  }
+  onDayClick(e) {
```

Comment:
please document this.

Reply:
the method name is clear enogh

Reply:
maybe is clear enough for you, not for me; the method name don't say anything on what are you doing inside.